.github/workflows/commit_message_checker.yml

```diff
@@ -0,0 +1,12 @@
+name: Commit Message Checker
+on:
+  push:
+    branches:
+      - feature
```

Comment:
It is sufficient to run this check on pull requests only. Similar to all other basic checks.